glide.yaml

```diff
@@ -25,5 +25,6 @@ import:
 - package: github.com/yuin/gopher-lua
 - package: gopkg.in/cheggaaa/pb.v1
 - package: gopkg.in/yaml.v2
+- package: github.com/btcsuite/btcd/wire
```

Comment:
Is it too big to reference the whole package? @raindust @RainFallsSilent

Reply:
Yes but we can improve it later when meta-data about transactions and UTXOs are well defined